src/languages/jsx.coffee

```diff
@@ -10,6 +10,7 @@ module.exports = {
   grammars: [
     "JSX"
     "JavaScript (JSX)"
+    "Babel ES6 JavaScript"
```

Comment:
Oh I didn't know about the priority. It would probably be better if I only specified the language inside the jsx hive. I guess in that way it would also pick up pretty-diff, which [now supports flow](https://github.com/prettydiff/prettydiff/issues/362), for all JavaScript files. If that is correct I will redo the PR?

 In my experience a lot of files, probably the majority, that have jsx content use a suffix of js. For example, [Facebook's draft.js editor](https://facebook.github.io/draft-js/) uses .js filetypes with jsx inside [here](https://facebook.github.io/draft-js/). I guess assuming a filetype is associated with content is a bit void. E.g. I also support typed flow which uses .js, and I support notion such as @decorators, class property shorthand, bind notation,..... which aren't in the JavaScript spec.